topo_processor/util/checksum.py

```diff
@@ -0,0 +1,7 @@
+import multihash
+
+
+def multihash_as_hex(file_path: str) -> str:
```

Comment:
Might be good to read the file in chunks. See https://github.com/linz/linz-data-lake/blob/master/data_lake/core/format/gdal_tile.py#L192

Reply:
what are the benefits of doing that?

Reply:
Will reduce amount of memory required. Good practise when dealing with largish files.